app/models/news_item.rb

```diff
@@ -53,6 +64,12 @@ def any_updates?
       end
     end
 
+    def cached_banner
```

Comment:
Looking where this is called, 
it looks it does not matter if it is cached or not,
I guess we could call this just `last_banner` (or maybe, even just `banner`)